src/i18n/languages/pt-BR.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @preserve
+ * Authors: Júlio C. Zuppa
+ * Last updated: Jan 12, 2018
+ *
+ * Description: Definition file for brazilian portuguese - Brazil language-country.
```

Comment:
Please change from: `Description: Definition file for brazilian portuguese - Brazil language-country.` to `Description: Definition file for Portuguese - Brazil language-country.`